.github/workflows/release.yml

```
@@ -366,6 +346,20 @@ jobs:
           draft: true
           prerelease: true
 
+      - name: Login to ACR
+        uses: docker/login-action@v1
```

```suggestion
        uses: docker/login-action@v1.6
```
Whenever possible we tend to use more specific versions in the actions, with the exception of official GitHub-provided actions (so these starting with `actions/*`).